### PR TITLE
win_say - fix up syntax and test issues (#45450)

### PR DESCRIPTION
**Pull request by jborean93**
(cherry picked from commit c9c141fb6a51d6b77274958a2340fa54754db692)

##### SUMMARY
Backport of https://github.com/ansible/ansible/pull/45450

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
win_say

##### ANSIBLE VERSION
```paste below
2.6
```

### CODE REVIEW

File: test/integration/targets/win_say/tasks/main.yml [hunk 1]
```diff
@@ -1,18 +1,24 @@
+# CI hosts don't have a valid Speech package so we rely on check mode for basic
+# sanity tests
```
Comment (mattclay) [inline test/integration/targets/win_say/tasks/main.yml:2]: Is it practical to install a speech package as part of the test? How much run time would it add to the test?

**Comment by jhawkesworth** [comment]
I don't think its worth it.  I have only seen the necessary parts in
'client' windows versions ( i.e. windows 10) and an audio output device is
needed, if I recall correctly.
Jon

**Comment by mattclay** [comment]
Assuming it can't be installed on servers (or isn't practical to do so), is there an easy way to detect if it's available? That would at least allow the full test to run manually on a client system.

File: test/integration/targets/win_say/tasks/main.yml (hunk 1 above)
Comment (jborean93) [reply test/integration/targets/win_say/tasks/main.yml:2]: I think it's more missing the actual audio capabilities to say the text when running on CI. Considering we have no way of verifying that it was said and all check mode does right now is skip the actual speech part I'm comfortable having it just cover this for now.

**Pull request merged**